quarkus/extensions/pom.xml

```diff
@@ -32,6 +32,14 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
```

Comment:
I can't see commons-lang being used anywhere. Is it needed?

Reply:
sorry, sneaked in by mistake